h/presenters.py

```diff
@@ -19,10 +19,12 @@ def _format_document_link(href, title, link_text, hostname):
         double-escaped.
 
     """
-    if hostname and hostname in link_text:
+    if href and hostname and hostname in link_text:
```

Comment:
Yes, I didn't want to go off and refactor this whole presenter, but I've renamed it to `host_or_filename`.